NOTICE

```diff
@@ -6,26 +6,6 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This product also includes the following third-party components:
 
- * ac_check_icu.m4 (http://autoconf-archive.cryp.to/ac_check_icu.html)
-
-   Copyright 2008, Akos Maroy <darkeye@tyrell.hu>
-
- * ac_check_curl.m4 (http://autoconf-archive.cryp.to/ac_check_curl.html)
-
-   Copyright 2008, Akos Maroy <darkeye@tyrell.hu>
-
- * pkg.m4 (http://http://pkg-config.freedesktop.org/wiki/)
-
-   Copyright 2004, Scott James Remnant <scott@netsplit.com>
-
- * jQuery (http://jquery.org/)
```

Comment:
Cannot remove, we still use bootstrap which still depends on jQuery.

Reply:
re-added, good catch